### PR TITLE
feat(javascript): add Fiber

### PR DESCRIPTION
**Pull request by gabrielcursino**
Please add a new JavaScript framework: [Fiber](https://github.com/ionited/fiber)

### CODE REVIEW

File: javascript/fiber/package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "@ionited/fiber": "^0.2.1"
```
Comment (waghanza) [inline javascript/fiber/package.json:3]:
```suggestion
    "@ionited/fiber": "~0.2.1"
```

File: javascript/fiber/config.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+framework:
+  website: ionited.io
+  version: 0.2.1
```
Comment (waghanza) [inline javascript/fiber/config.yaml:3]:
```suggestion
  version: 0.2
```

**Review by waghanza — COMMENTED** [review]
error since we use `alpine`
```javascript
Error: This version of uWS.js supports only Node.js LTS versions 16, 18 and 20 on (glibc) Linux, macOS and Windows, on Tier 1 platforms (https://github.com/nodejs/node/blob/master/BUILDING.md#platform-list).
Error: Error loading shared library ld-linux-x86-64.so.2: No such file or directory (needed by /usr/src/app/node_modules/uWebSockets.js/uws_linux_x64_108.node)
    at /usr/src/app/node_modules/uWebSockets.js/uws.js:22:9
    at Object.<anonymous> (/usr/src/app/node_modules/uWebSockets.js/uws.js:24:3)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1310:10)
    at Module.load (node:internal/modules/cjs/loader:1119:32)
    at Module._load (node:internal/modules/cjs/loader:960:12)
    at Module.require (node:internal/modules/cjs/loader:1143:19)
    at Hook._require.Module.require (/usr/local/lib/node_modules/pm2/node_modules/require-in-the-middle/index.js:101:39)
    at require (node:internal/modules/cjs/helpers:121:18)
    at Object.<anonymous> (/usr/src/app/node_modules/@ionited/fiber/dist/app.js:4:26)
```

**Comment by gabrielcursino** [comment]
How can i help?

I found this on Stack Overflow: https://stackoverflow.com/questions/66963068/docker-alpine-executable-binary-not-found-even-if-in-path

**Comment by waghanza** [comment]
I think https://github.com/ionited/fiber/blob/3019f6b375a9432d2a7e53e7dda5ed173368f698/package.json#L29 is incorrect, you should declare
```json
"uWebSockets.js": "github:uNetworking/uWebSockets.js#v20.31.0"
```

**Comment by gabrielcursino** [comment]
Hello @waghanza, i've updated the package!

File: javascript/fiber/config.yaml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+framework:
+  website: ionited.io
+  version: 0.2
```
Comment (waghanza) [inline javascript/fiber/config.yaml:3]:
```suggestion
  version: 0.2
  engines:
    - uwebsockets
```

**Review by waghanza — CHANGES_REQUESTED** [review]

**Comment by waghanza** [comment]
I will update results for monday @gabrielcursino, better not to miss this train 😛

**Review by waghanza — APPROVED** [review]

**Pull request merged**